Week1/MAKEME.md

```diff
@@ -27,20 +27,22 @@ Write a JavaScript file (to be executed with Node.js) that creates and makes a c
 > Before working on the exercise, first get a working version of the [example](https://www.npmjs.com/package/mysql#introduction) going. This will help you understand the basic structure of making a database connection and query. When doing this, make sure you replace the `user`, `password` and `database` values with the ones you've created for this module: `hyfuser`, `hyfpassword` and `user_db`.
```

Comment:
This part can be omitted because this is not part of the homework. LESSONPLAN covers the use of db connection.